data/org.freedesktop.portal.Notification.xml

```diff
@@ -165,6 +167,7 @@
     <signal name="ActionInvoked">
       <arg type="s" name="id"/>
       <arg type="s" name="action"/>
+      <arg type="s" name="activation_token"/>
```

Comment:
Doesn't this break all existing users of this signal?

To avoid that, we can create a `ActionInvoked2`, and while at it, make it pass a `a{sv}` to make things a bit more future proof.

Reply:
> Doesn't this break all existing users of this signal?

Yes it does. (</dbus-maintainer>)

Reply:
Yeah, indeed, reason why I bumped the protocol version.

I was quickly discussing this with @matthiasclasen, but indeed another versioned signal could be added instead.

But my point was that clients should check on version before.

@jadahl I agree could be just an `a{sv}` platform data though.

Reply:
Apps that didn't update will not check the version though, as from their perspective, version 2 doesn't exist. And as there is no "binding" that lets the portal send the right signature to old applications, they would just fail / crash.

Reply:
This is fixed now, added the platform-data dict.